### PR TITLE
Added skydive-project deployment

### PR DESCRIPTION
**Pull request by SchSeba**
**What this PR does / why we need it**:
Allow the user to deploy [skydive](http://skydive.network/)

**Release note**:
```release-note
Deploy skydive-project into the cluster
```

---
This change is [Reviewable](https://reviewable.io/reviews/kubevirt/kubevirt-ansible/415)

### CODE REVIEW

**Comment by SchSeba** [comment]
ci test please

File: roles/skydive/tasks/provision.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+---
+- name: oc Command
+  set_fact:
+    cluster_command: oc
+  when: cluster=="openshift"
```
Comment (rthallisey) [inline roles/skydive/tasks/provision.yml]: you shouldn't need to specify a client.   Use kubectl by default and only use `oc` if `when: cluster=="openshift"` is used.

**Comment by rthallisey** [comment]
@leongold @SchSeba also needs a rebase

File: roles/skydive/tasks/deprovision.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+- name: openshift cni config
```
Comment (phoracek) [inline roles/skydive/tasks/deprovision.yml]: how is this related? is that because of missing rebase?

**Comment by SchSeba** [comment]
ci test please

**Comment by SchSeba** [comment]
@rthallisey I change as requested but please check that now we need two deploy stages.
One for openshift because we use template object kind so the command must be `oc command` , and the second one for a regular `kubectl`

**Comment by gbenhaim** [comment]
@rthallisey Are you ok with this change?

**Comment by SchSeba** [comment]
ci test please

**Comment by rthallisey** [comment]
ci test please

**Comment by SchSeba** [comment]
@rthallisey no need for this PR any more replaced by https://github.com/kubevirt/kubevirt-ansible/pull/438
